data/projects/vulture.md

```diff
@@ -42,16 +42,19 @@ References:
 
```

Comment:
You could add `use vulture to report dead code for popular Python projects on github` to the preparation phase (see https://github.com/jendrikseipp/vulture/issues/38).